front-end/src/renderer/components/Transaction/Create/NodeCreate/NodeFormData.vue

```diff
@@ -72,6 +81,83 @@ function handleDeleteEndpoint(index: number, key: 'gossipEndpoints' | 'serviceEn
   });
 }
 
+async function handleInputGossipCert(e: Event) {
+  const target = e.target as HTMLInputElement;
+  await handleUpdateGossipCert(target.value);
+}
+
+async function handleUpdateGossipCert(str: string) {
+  const publicKey = str.split('-----')[2];
+  if (!str) {
```

Comment:
Should it be `if (!publicKey)`, instead of `if (!str)`?
You accessing the third element of the array, which may be undefined if the `str` is not in the correct format